src/main/java/org/terasology/signalling/blockFamily/SignalCableBlockFamilyFactory.java

```diff
@@ -100,7 +100,14 @@
     
     public SignalCableBlockFamilyFactory(){
     }
-    
+
+    /**
+     * Creates a new {@link BlockFamily} with a given set of definitions.
+     *
+     * @param definition
+     * @param blockBuilder
+     * @return A new {@link SignalUpdateFamily} with the given definitions, always non-null.
+     */
```

Comment:
You appear to have missed the @param tags here. Should be a fairly simple thing to add